source/advocating.html.erb.md

```diff
@@ -11,41 +11,41 @@ status: editors-draft
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Develop accessibility business case
+## Develop an accessibility business case
 
 [ ... Tip description goes here ... ]
 
 {:.attach_permalink}
-## Secure management support for accessibility activities
+## Address management to support accessibility activities
```

Comment:
For discussion. I think "secure" is a tall order and not always feasible for an advocate (as opposed to an accessibility manager or such).